clonelists/Sony - PlayStation 2.json

```diff
@@ -2960,6 +2972,15 @@
 		"XIII": [
 			["XIII - Daitouryou o Koroshita Otoko", 1]
 		],
+		"Yakuza": [
+			["Ryuu ga Gotoku", 1]
+		],
+		"Yakuza 2": [
```

Comment:
JSON only accepts unique keys, or it's considered invalid. Here you've got Yakuza 2 twice, which will throw an error.

The way to get around it when you've got multiple discs in one region, and only one disc in another, is to use the 0 priority like so:

"Ryuu ga Gotoku 2 (Disc 1)": [
    ["Yakuza 2", 0]
],
"Ryuu ga Gotoku 2 (Disc 2)": [
    ["Yakuza 2", 0]
],

Reply:
Learning something new every time ^^;
Thanks for your patience !